source/ncengine/graphics2/diligent/ShaderFactory.h

```diff
@@ -46,7 +46,7 @@ class ShaderFactory
     private:
         Diligent::IRenderDevice* m_device;
         Diligent::RefCntAutoPtr<Diligent::IShaderSourceInputStreamFactory> m_streamFactory;
-        std::string_view m_shadersPath;
+        std::string m_shadersPath;
```

Comment:
As a `string_view` this pointed inside the global config object. Seemed like a UB timebomb if we ever change how that is stored.

Reply:
GREAT CATCH!